lwatools/file_tools/results.py

```diff
@@ -0,0 +1,48 @@
+import h5py
+from lsl.common import stations
+
+from lwatools.utils import known_transmitters
+
+def setup_results_h5(h5_fname, tbnf, transmitter, tx_freq, valid_ants, n_baselines, fft_len, use_pfb, use_pol, integration_length, opt_method, res_function_name, station=stations.lwasv):
```

Comment:
Maybe something like `build_output_structure`. Nominally I would call our results what we get from the output.

Reply:
gonna go with `build_output_file` because it returns a file object

Reply:
also i'll rename the file to outputs.py